### PR TITLE
release 2.9.0

### PR DESCRIPTION
**Pull request by badenerb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,25 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/lsst-it/puppet-ccs_sal/compare/v2.8.0...v2.8.1)
+
 ## [v2.8.0](https://github.com/lsst-it/puppet-ccs_sal/tree/v2.8.0) (2024-08-29)
 
 [Full Changelog](https://github.com/lsst-it/puppet-ccs_sal/compare/v2.7.0...v2.8.0)
 
 **Implemented enhancements:**
 
+- Version 2.8.0 [\#49](https://github.com/lsst-it/puppet-ccs_sal/pull/49) ([glennmorris](https://github.com/glennmorris))
```
Comment (jhoblitt) [inline CHANGELOG.md:25]: Could you change the label to `skip-changelog` for this PR and regenerate the changelog?  Someone reading the log probably already knows that the 2.8.0 release is the 2.8.0 release. :)

Comment (badenerb) [reply CHANGELOG.md:25]: image
I did add that label... perhaps I messed it up somehow 😆

Comment (jhoblitt) [reply CHANGELOG.md:25]: The label on this PR is correct but on #49 it is image when it should be `skip-changelog`.

Comment (badenerb) [reply CHANGELOG.md]: Fixed this by changing the tag on [#49 ](url)and regenerated the changelog

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**